lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/Media.java

```diff
@@ -0,0 +1,9 @@
+package com.codedifferently.lesson10.library;
+
+public interface Media {
```

Comment:
I'd like to see you integrate this with the `Library` class. You'll also need to update the test.